modulefiles/noaacloud-run.intel.lua

```diff
@@ -1,24 +1,20 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/contrib/spack-stack-rocky8/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/contrib/spack-stack-rocky8/spack-stack-1.9.1/envs/ue-oneapi-2024.2.1/install/modulefiles/Core")
```

Comment:
Is 1.9.2 available on the cloud?  The weather model is headed there and it would be useful to move to it if possible.